src/control/costmap_queue/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
Let's push the source code down one level in the directory hierarchy. For example, under navigation2/src/control/nav2_dwb_controller. There may be opportunities for code sharing among the controllers, but we can factor that out later.